frontend/src/modules/machinery/components/Machinery.js

```diff
@@ -46,7 +57,12 @@ export default class Machinery extends React.Component<Props> {
             <div className="search-wrapper clearfix">
                 <div className="icon fa fa-search"></div>
                 <Localized id="machinery-machinery-search-placeholder" attrs={{ placeholder: true }}>
-                    <input type="search" autoComplete="off" placeholder="Type to search machinery" />
+                    <input
+                        type="search"
+                        autoComplete="off"
+                        placeholder="Type to search machinery"
+                        onKeyDown={ this.handleShortcuts }
+                    />
```

Comment:
I suspect this would be cleaner if it was put inside a form and we used the `onSubmit` event of the form to handle the custom search instead. Less hackish, more standard, probably more accessible too.